bvm/ballerina-core/src/main/java/org/ballerinalang/core/model/types/BStructureType.java

```diff
@@ -64,11 +64,11 @@ public void setFieldTypeCount(int[] fieldCount) {
         this.fieldTypeCount = fieldCount;
     }
 
-    public BAttachedFunction[] getAttachedFunctions() {
+    public BAttachedFunction[] getMemberFunctionTypes() {
         return attachedFunctions;
     }
 
-    public void setAttachedFunctions(BAttachedFunction[] attachedFunctions) {
+    public void setMemberFunctionTypes(BAttachedFunction[] attachedFunctions) {
```

Comment:
Shall we rename the parameter too?

Also, should we rename the methods to `xxxMemberMethodTypes` instead of `xxxMemberFunctionTypes`?

Reply:
I think this should be something like `setMethodTypes` @MaryamZi @hasithaa WDY think about that name?

Reply:
+1